maxscale-kafka-adapter/CMakeLists.txt

```diff
@@ -52,41 +52,14 @@ add_subdirectory(src)
 install(FILES "README.md"
     "LICENSE"
     DESTINATION "${CMAKE_INSTALL_DOCDIR}"
+    COMPONENT mcskafka
     )
 
-option(RPM "Build an RPM" OFF)
-
-if (RPM)
-    SET(CPACK_GENERATOR "RPM")
-    SET(CPACK_RPM_PACKAGE_VERSION ${VERSION})
-    SET(CPACK_RPM_PACKAGE_RELEASE "1")
-    SET(CPACK_RPM_PACKAGE_NAME "mariadb-columnstore-kafka-adapters")
-    SET(ENGINE_ARCH "x86_64")
-    SET(CPACK_PACKAGE_FILE_NAME "${CPACK_RPM_PACKAGE_NAME}-${CPACK_RPM_PACKAGE_VERSION}-${CPACK_RPM_PACKAGE_RELEASE}-${ENGINE_ARCH}-${RPM}")
-
-    include(CPack)
-endif()
-
-option(DEB "Build a DEB" OFF)
-
-if (DEB)
-    SET(CPACK_GENERATOR "DEB")
-    SET(CPACK_DEBIAN_PACKAGE_VERSION ${VERSION})
-    SET(CPACK_DEBIAN_PACKAGE_RELEASE "1")
-    SET(CPACK_DEBIAN_PACKAGE_NAME "mariadb-columnstore-kafka-adapters")
-    SET(CPACK_PACKAGE_CONTACT "MariaDB Corporation")
-    SET(ENGINE_ARCH "x86_64")
-    SET(CPACK_DEBIAN_PACKAGE_SHLIBDEPS ON)
-    SET(CPACK_PACKAGE_FILE_NAME "${CPACK_DEBIAN_PACKAGE_NAME}-${CPACK_DEBIAN_PACKAGE_VERSION}-${CPACK_DEBIAN_PACKAGE_RELEASE}-${ENGINE_ARCH}-${DEB}")
-    include(CPack)
-endif()
-
 # uninstall target
 configure_file(
     "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/cmake_uninstall.cmake.in"
     "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
     IMMEDIATE @ONLY)
 
-add_custom_target(uninstall
+add_custom_target(uninstall_mcskafka
```

Comment:
The original uninstall command should be moved to the top-level CMake file to have both `make install` and `make uninstall`. The project specific ones can be removed.